src/main/java/de/tum/in/www1/artemis/service/connectors/bitbucket/dto/BitbucketCloneDTO.java

```diff
@@ -30,6 +36,15 @@ public void setCloneDetails(CloneDetailsDTO cloneDetails) {
         this.cloneDetails = cloneDetails;
     }
 
+    /**
+     * Creating a description for the console log
+     * @return description with name and project key
+     */
+    @Override
+    public String toString() {
+        return "BitbucketCloneDTO{" + "name='" + name + '\'' + ", cloneDetails=" + cloneDetails.toString() + '}';
+    }
+
     public static final class CloneDetailsDTO {
 
```

Comment:
Maybe add here also a constructor, like for `BitbucketCloneDTO` :)

Reply:
yes thanks!